src/parser/TraverseSentences.h

```diff
@@ -406,14 +406,18 @@ class FetchEdgesSentence final : public Sentence {
         return yieldClause_.get();
     }
 
-    std::string* edge() const {
-        return edge_.get();
+    const std::string* edge() const {
```

Comment:
the edge also support multi labels, the `edge()` function is needed to modify `edges()`

Reply:
Don't support in fact.